docs/core/EigenPodManager.md

```diff
@@ -192,7 +184,7 @@ For each validator the Pod Owner wants to verify, the Pod Owner must supply:
     * `VALIDATOR_STATUS` moves from `INACTIVE` to `ACTIVE`
     * `validatorIndex` is recorded
     * `mostRecentBalanceUpdateTimestamp` is set to the `oracleTimestamp` used to fetch the beacon block root
-    * `restakedBalanceGwei` is set to `_calculateRestakedBalanceGwei(effectiveBalance)`
+    * `restakedBalanceGwei` is set to the validator's balance
```

Comment:
in this particular instance we use the effective (rather than actual) balance, right?